targets/newtonsoft-json.props

```diff
@@ -12,5 +12,6 @@
     <BuildPackagesCommand>$(DotNetCommand) pack $(NewtonsoftJsonProjectPath) $(DotNetCommandArguments)</BuildPackagesCommand>
     <CleanCommand>$(DotNetCommand) clean $(NewtonsoftJsonProjectPath) $(DotNetCommandArguments)</CleanCommand>
     <PackagesOutput>$(NewtonsoftJsonDirectory)bin/$(Configuration)/</PackagesOutput>
+    <RepoApiNotImplemented>true</RepoApiNotImplemented>
```

Comment:
Rather than a negative property i think we should consider a positive one. 

``` xml
<RepoApiImplemented>true</RepoApiImplemented>
```

Reply:
Why not default to true, when unset, and then force all of the uncompliant projects to override to false? That is the same amount of churn as this change. It also has the benefit that the code reads much nicer and the final state is a lot cleaner

Reply:
Since this is just a temporary parameter (the end goal is that this property and the others go away when all repos are compliant), I don't think it matters too much either way.

@jaredpar is right that the code does read nicer when `Condition="'$(RepoApiImplemented)' == 'true'"` and `Condition="'$(RepoApiImplemented)' != 'true'"`.  I'll flip it to that for now.